catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/AttributeRegistry.java

```diff
@@ -32,7 +32,7 @@
      * @throws IllegalArgumentException if {@code attributeDescriptor} or
      *                                  {@link AttributeDescriptor#getName()} is null
      */
-    boolean registerAttribute(AttributeDescriptor attributeDescriptor);
+    boolean register(AttributeDescriptor attributeDescriptor);
```

Comment:
I wanted to highlight the interface changes here.  This is marked as an experimental interface so this is allowed, but I know there are already places in the Catalog UI this is being used @rzwiefel